package/com.unity.formats.usd/Samples/UsdTimelinePlayable/UsdTimelineRecorderExample.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Playables;
+
+namespace Unity.Formats.USD.Examples
+{
+    [ExecuteAlways]
+    public class UsdTimelineRecorderExample : MonoBehaviour
+    {
+        [HideInInspector]
+        public bool m_playableDirectorExists = false;
+
+        private void Update()
+        {
+            m_playableDirectorExists = this.GetComponent<PlayableDirector>() != null;
```

Comment:
I've mentioned above that I don't think we should be doing this as part of a sample, but just a bit of extra advice here- We shouldn't do GetComponent in an Update loop, even if it is 'just' in the Editor. GetComponent can be very costly and as the Update loop runs every frame this can be harmful for performance, so we don't want to encourage it to users :)